MAUI/NavigationDrawer/Navigation-Pane-Sides.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title: Configuring The Drawer In .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about Configuring The Drawer In Different Sides support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Configuring The Drawer In Different Sides in .NET MAUI Navigation Drawer
+
+The supplemental pane can be drawn in and out from all four sides. [`Position`]() property is used to change the side of the pane and the values are
```

Comment:
The supplemental pane can be pulled in and out from all four sides. The `Position` property is used to change the side of the pane, and the options are,

Reply:
Changed